stk/calculators/optimization/macromodel.py

```diff
@@ -541,10 +542,9 @@ class MacroModelForceField(_MacroModel):
     Attributes
     ----------
     restricted : :class:`bool`
-        If ``False`` then all bonds are optimized, not just the ones
-        created during macromolecular assembly. If ``True`` then an
-        optimization is performed only on the bonds added during
-        molecular assembly.
+        If ``True`` then an optimization is performed only on the bonds
+        added by :meth:`~.Topology.construct`. If ``False`` then all
+        bonds are optimized.
```

Comment:
I am thinking about how to make this more clear. I.e. its not just the bond potentials that are allowed to relax, it is also the angles and torsions around those bonds? is that correct?